packages/console/package.json

```diff
@@ -43,7 +43,7 @@
     "lodash.throttle": "^4.1.1",
     "memoize-one": "^5.1.1",
     "memoizee": "^0.4.15",
-    "monaco-editor": "^0.27.0",
+    "monaco-editor": "^0.34.0",
```

Comment:
You checked the changelog for any breaking changes between versions?

Reply:
I did - there were a few breaking changes...
- `KeyMod` location changed, it was no longer in standaloneEditor
- `getModeId()` changed to `getLanguageId()`
- KeyCodes were renamed from `KEY_A` to `KeyA` etc
I think that was it